app/controllers/log_controller.rb

```diff
@@ -0,0 +1,4 @@
+class LogController < ApplicationController
+    def entries
+    end
```

Comment:
Indentation should be 2 spaces, this needs formatting

Reply:
Thank you for the information, i changed it